pages/community/roadmap.mdx

```diff
@@ -0,0 +1,95 @@
+- During active development, there are no breaking changes within the compiler (no changes in runtime representation, behavior of the build system, configuration names etc.). 
+- No removal of language features
+- During the active development phase, there may be non-breaking minor releases (e.g. v10.1.0). For example:
+  - Opt-in configuration flag `stdlib-future` to disable OCaml related stdlib code (for writing future proof code)
+  - Stdlib additions / Library enhancements
+- Regular bugfix releases for the compiler, build system, stdlib, etc.
+- Docs will be synced up with the latest stable release
+- Whenever a new stable release is being published, the previous stable release reaches end-of-life (EOL), and therefore will only receive bugfix updates in a best effort service. After 1 year LTS, we expect the retired version to be pretty stable though, so in practise, users will be able to stay on that version indefinitely until they decide to upgrade to the newest stable
+
+### Unstable / experimental release:
+
+- Uneven number (v11, v13, v15,....)
+- Development phase of around 6 months. After that it will potentially turn into the next stable release (e.g. v11 -> v12)
+- Introduces a planned set of breaking changes based on the previous stable release
+- They are supposed to be a stepping stone to reach more ambitious goals in a next stable release
+- The list of breaking changes will be discussed before the work is being started
+- No guaranteed stability / support during unstable development. The compiler toolchain will be reliable as always, but the proposed feature set might change
+- Docs might or might not be available during experimental changes
+
+### How to install relevant releases:
+
+- Latest stable: `npm install rescript` (aka `@latest`)
+- Latest unstable (in case we are in an “unstable” dev phase): `npm install rescript@next`
+
+### Estimated Release plan for v10 - v15
+
+<Image withShadow={false} src="/static/img/roadmap_2021.jpg" caption="Release cycle for the upcoming versions"/>
+
+### Feature Discussions and Proposals
+
+All our discussions and proposals for the roadmap happen in the [ReScript Forum](https://forum.rescript-lang.org).
+
+## Stable Release v10
+
+**Planned Changes for v10:**
+
+- Add M1 support
+- Add support for unicode strings (e.g `"Hallöle"` will work like in JavaScript)
+- (breaking) Remove OCaml’s Format module (should have no impact on the majority of ReScript users)
+- Infix operator support (only for a dedicated set of math operators `+`, `-`, `-.`, `/`, `%`, etc.)
+- Docstring syntax support (`/** */` instead of `@ocaml.doc("My docstring")`)
+- Convert stdlib files (`Js`, `Belt`, etc) to ReScript syntax
+- Some other less prominent enhancements (will be mentioned in the Changelog later on)
+- We will eventually announce additional breaking changes in late 2021 if necessary
```

Comment:
Maybe these should link to the corresponding GitHub issues/epics? That way it is easier to look up the current status of these changes and/or contribute to them.

Reply:
added

Reply:
Awesome!